### PR TITLE
Added User-Agent String to Traffic Router Log Output

### PR DESCRIPTION
**Pull request by ARMmaster17**
## What does this PR (Pull Request) do?
Requested by @smalenfant, adds User-Agent header string to log output from Traffic Router for each request.

- [x] This PR is not related to any Issue

## Which Traffic Control components are affected by this PR?

- Traffic Router

## What is the best way to verify this PR?
Pull and run CDN-in-a-Box and make a request against traffic router using the built-in VNC container (or your own custom solution). Check the console output of Traffic Router from the request and verify the that a field exists similar to `uas="Mozilla/5.0 (X11; Linux x86_64; rv:68.0) Gecko/20100101 Firefox/68.0"`.

## The following criteria are ALL met by this PR

- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added Traffic Monitor: Support stats over http CSV output. Officially supported in ATS 9.0 unless backported by users. Users must also include `system_stats.so` when using stats over http in order to keep all the same functionality (and included stats) that astats_over_http provides.
 - Added ability for Traffic Monitor to determine health of cache based on interface data and aggregate data. Using the new `stats_over_http` `health.polling.format` value that allows monitoring of multiple interfaces will first require that *all* Traffic Monitors monitoring the affected cache server be upgraded.
 - Traffic Ops, Traffic Ops ORT, Traffic Monitor, Traffic Stats, and Grove are now compiled using Go version 1.15.
-- Added `--traffic_ops_insecure=<0|1>` optional option to traffic_ops_ort.pl
```
Comment (zrhoffman) [inline CHANGELOG.md:59]: We still want this changelog message

**Review by zrhoffman — CHANGES_REQUESTED** [review]
Just 1 comment about the changelog, everything else looks great.

File: CHANGELOG.md (hunk 1 above)
Comment (ARMmaster17) [reply CHANGELOG.md:59]: Oops! Should be fixed now.

**Review by zrhoffman — APPROVED** [review]
* Unit tests pass
* Integration tests pass
* External tests pass

Confirmed the user agent shows up as expected in the logs.

**Pull request merged**